### PR TITLE
Ui home backend integration

### PR DESCRIPTION
**Pull request by jatinKamnani**

### CODE REVIEW

File: public/js/login.js [hunk 1]
```diff
@@ -15,6 +17,7 @@ $(document).ready(function () {
         };
 
         $.post("/api/login", JSON.stringify(loginCreds), function (data) {
+            localStorage.setItem("login", "TRUE");
```
Comment (uchanchlani) [inline public/js/login.js]: What if the login password is incorrect or the session expires? 
We need to invalidate the local storage.

Comment (jatinKamnani) [reply public/js/login.js]: you are not resetting my cookie when I logout, so had to do this. This was one thing i wanted to discuss

Comment (jatinKamnani) [reply public/js/login.js]: I can do it based on cookie once on logout you empty my cookie

Comment (jatinKamnani) [reply public/js/login.js]: cool. Didnt know about this api.

Comment (uchanchlani) [reply public/js/login.js]: No, the cookie gets removed. The /api/logout call removes the cookie.

Comment (jatinKamnani) [reply public/js/login.js]: I checked it, the cookie was having a value. Nevermind will do that later now

**Comment by uchanchlani** [comment]
Dude, make the script requests over https, else it won't be loaded in the production environment. I am facing that problem now.

**Pull request merged**